frontend/src/containers/ResetPassword/ResetPasswordPage.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from "react";
+import LogInPageHeader from "../../components/LogInPageHeader/LogInPageHeader";
+import NewPassword from "../../components/NewPassword/NewPassword";
+import Footer from "../../components/Footer/Footer";
+import "./ResetPasswordPage.css";
+
+class resetPasswordPage extends Component {
```

Comment:
I think we want to start component names (and classes generally) with capital letters.